### DISCUSSION TITLE
Maybe error in documentation:  npm install after npx nuxi@latest init <project-name> is not necessary

### DISCUSSION THREAD

**Discussion by RollingTL** [nuxt/nuxt#26588] [General]
Docs https://nuxt.com/docs/getting-started/installation tell us:

> npx nuxi@latest init <project-name>
> Open your project folder in Visual Studio Code:
> 
> Terminal
> code <project-name>
> Or change directory into your new project from your terminal:
> cd <project-name>
> Install the dependencies:
> 
> npm install

But everything is installed after npx nuxi@latest init finished.  I am not sure but it looks like Docs error.

**Comment by TheAlexLichter** [comment]
PR welcome 👍🏻

**Reply by RollingTL** [reply]
Sorry. Never done it. Need time to learn the process. At the moment have no time to learn it. So pls do it yourself.

**Reply by TheAlexLichter** [reply]
@RollingTL No worries at all! Happy to help you (and any future reader) with that as it is less than a few minutes of work ☺️

1. Open the URL you linked before
2. On the right side (desktop) there is an "edit this page" link (see image below). This will open the page as markdown
3. Make your changes
4. Click "Commit Changes" on top
5. Use your own fork and branch, then click "propose changes"
6. The "open a pull request" window will be opened. Fill out the questions (will be quick) and click "Create Pull Request"

<img width="1228" alt="image" src="https://github.com/nuxt/nuxt/assets/640208/fbba32f9-838f-4c2a-a0a4-ed9fc6ab596a">


---

And yes, while I could do it myself easily, it can help people starting to contribute to Open Source ☺️

**Reply by RollingTL** [reply]
Thanks for instructions.

Done my first pull request. 

https://github.com/nuxt/nuxt/pull/26676
Hope its ok.

**Reply by RollingTL** [reply]
Cool. I am contributor!
